### PR TITLE
Create cbcd_contra

### PR DESCRIPTION
**Pull request by charlieturner**
Two new dances. "CBCD" is a working title

### CODE REVIEW

File: events/uk/cbcd_contra [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    end: 2022-10-22T23:00:00+01:00
+    country: UK
+    city: Cambridge
+    styles: [contra]
+    workshop: false
+    social: true
+    bands:
+      - Flying Cat
+      - Matt Cudby
+      - 'Flying Matt'
```
Comment (qwandor) [inline events/uk/cbcd_contra:15]: Are these actually all one band, or three different bands playing separately?

File: events/uk/cbcd_contra [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    callers:
+      - Adam Hughes
+    price: £7-£15
+
+- name: Contra Dance in Cambridge
+    links:
+      - "https://www.flyingcat.dance/gigs/newcontras"
+    start: 2022-11-19T19:45:00+01:00
+    end: 2022-11-19T23:00:00+01:00
```
Comment (qwandor) [inline events/uk/cbcd_contra:24]: I guess this should be +00:00.

**Review by qwandor — COMMENTED** [review]
Thanks! Can you change the filename to end in `.yaml`, and have the file start with the same first two lines as the other files? You can also add an organisation field if you like, to show up in the list at https://folkdance.page/organisations.

File: events/uk/cbcd_contra (hunk 1 above)
Comment (charlieturner) [reply events/uk/cbcd_contra:15]: One band, playing under "Flying Matt", but I want "Flying Cat" to get their credit as I expect "Flying Matt" to be a one-time thing.

**Pull request merged**